raiden/exceptions.py

```diff
@@ -269,8 +269,8 @@ class ReplacementTransactionUnderpriced(RaidenError):
     """Raised when a replacement transaction is rejected by the blockchain"""
 
 
-class TransactionAlreadyPending(RaidenUnrecoverableError):
```

Comment:
I had to remove this exception since it cannot be consistently raised when parity is used. (Not with this version of the code, I'm delaying this until #2088)